Backpack/HorizontalNavigation/Classes/BPKHorizontalNavigation.m

```diff
@@ -257,6 +259,7 @@ - (void)repopulateStackview {
     // Remove all current arranged views:
     for (UIView *subView in self.stackView.arrangedSubviews) {
         [self.stackView removeArrangedSubview:subView];
+        [subView removeFromSuperview];
```

Comment:
There are two steps needed to remove a subview from a `UIStackView`, this one was missing